src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,12 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery ballFirst = new Lottery();
+        Lottery ballSecond = new Lottery();
+        Lottery ballThird = new Lottery();
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
```
тут достатньо одної лотереї, вона може створювати багато рандомних кульок